doc/dev/packaging.md

```diff
@@ -1,26 +1,25 @@
 # Azure packaging
 
-This article describes how to declare setup.py and all packaging information for packages inside the `azure` namespace
+This article describes the recommendation on how to define namespace packaging to release a package inside the `azure` namespace. Being inside the `azure` namespace meaning you have a service `myservice` that you want to import using:
+```python
+import azure.myservice
+```
 
 Namespace packaging is complicated in Python, here's a few reading if you still doubt it:
 - https://packaging.python.org/guides/packaging-namespace-packages/
 - https://www.python.org/dev/peps/pep-0420/
 - https://github.com/pypa/sample-namespace-packages
 
-This article describes the recommendation on how to define namespace packaging to release a package inside the `azure` namespace. Being inside the `azure` namespace meaning you have a service `myservice` that you want to import using:
-```python
-import azure.myservice
-```
-
 Note:
-- This article is not about setup.py or setup.cfg or the right way to *write* the packaging, it's about what instructions you should use to achieve this. If you are fluent in setuptools, and prefer to write the suggestions in setup.cfg and not in setup.py, this is not a concern.
+While this article provides example using setup.py, there is no problem with achieving it with setup.cfg or other methods, as long as the constraints on the final wheels/sdist are met.
```

Comment:
"...provides an example using setup.py, this can also be achieved with setup.cfg, or other methods..."